examples/ingest/sharepoint/ingest.sh

```diff
@@ -12,6 +12,8 @@
 # To get the credentials for your Sharepoint app, follow these steps:
 # https://github.com/vgrem/Office365-REST-Python-Client/wiki/How-to-connect-to-SharePoint-Online-and-and-SharePoint-2013-2016-2019-on-premises--with-app-principal
 
+# To setup your application and obtain permissions related variables (--permissions-application-id, --permissions-client-cred, --permissions-tenant), follow these steps:
+# https://tsmatz.wordpress.com/2016/10/07/application-permission-with-v2-endpoint-and-microsoft-graph
```

Comment:
since this is from a random wordpress site, can you archive these instructions somewhere internally on our end? want to make sure we don't lose this if that site disappears.

Reply:
I've added the contents to our google drive and shared with you now 👍